src/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater/WCFUpdateStep.cs

```diff
@@ -0,0 +1,226 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater
+{
+    [ApplicableLanguage(Language.CSharp)]
+    public class WCFUpdateStep : UpgradeStep
+    {
+        public override string Title => "Update WCF service to CoreWCF.";
+
+        public override string Description => "Update WCF service to use CoreWCF services. For more information about CoreWCF, please go to: https://github.com/CoreWCF/CoreWCF";
+
+        public override string Id => WellKnownStepIds.WCFUpdateStepId;
+
+        private ConfigUpdater? _configUpdater;
+
+        private SourceCodeUpdater? _sourceCodeUpdater;
+
+        private PackageUpdater? _packageUpdater;
+
+        private List<SourceCodeUpdater>? _directiveUpdaters;
+
+        private Dictionary<string, IEnumerable<string>>? _path;
+
+        public override IEnumerable<string> DependsOn { get; } = new[]
+        {
+            // Project should be backed up before changing source
+            WellKnownStepIds.BackupStepId,
+
+            // Template files should be added prior to changing source (since some code fixers will change added templates)
+            WellKnownStepIds.TemplateInserterStepId,
+
+            // Project should have correct TFM
+            WellKnownStepIds.SetTFMStepId,
+
+            // Project's config file, source code, and project file need to be updated first before adding CoreWCF services
+            WellKnownStepIds.ConfigUpdaterStepId,
```

Comment:
Does the ConfigUpdater belong here? I'd have thought you'd want to run before it in order to remove the system.ServiceModel section.